sdk/monitor/azure-monitor-opentelemetry-exporter/README.md

```diff
@@ -29,50 +29,87 @@ In order to export telemetry data to Azure Monitor, you will need the instrument
 
-##### Setup OpenTelemetry SDK to work with Azure Monitor exporter
-```java readme-sample-setupExporter
-// Create Azure Monitor exporter and initialize OpenTelemetry SDK
-// This should be done just once when application starts up
+```java readme-sample-autoconfigure-env-variable
 AutoConfiguredOpenTelemetrySdkBuilder sdkBuilder = AutoConfiguredOpenTelemetrySdk.builder();
-
-new AzureMonitorExporterBuilder()
-    .connectionString("{connection-string}")
-    .install(sdkBuilder);
-
+AzureMonitor.customize(sdkBuilder);
 OpenTelemetry openTelemetry = sdkBuilder.build().getOpenTelemetrySdk();
+```
 
-Tracer tracer = openTelemetry.getTracer("Sample");
+You can also se the connection string in the code:
+```java readme-sample-autoconfigure
+AutoConfiguredOpenTelemetrySdkBuilder sdkBuilder = AutoConfiguredOpenTelemetrySdk.builder();
+AzureMonitor.customize(sdkBuilder, "{connection-string}");
+OpenTelemetry openTelemetry = sdkBuilder.build().getOpenTelemetrySdk();
 ```
 
-##### Create spans
+## Examples
+
+The following sections provide code samples using the OpenTelemetry Azure Monitor Exporter client library and OpenTelemetry API.
+
+
+The following example shows how create a span:
+
+```java readme-sample-create-span
+AutoConfiguredOpenTelemetrySdkBuilder otelSdkBuilder = AutoConfiguredOpenTelemetrySdk.builder();
+
+AzureMonitor.customize(otelSdkBuilder, "{connection-string}");
 
-```java readme-sample-createSpans
-// Make service calls by adding new parent spans
-ConfigurationClient client = new ConfigurationClientBuilder()
-    .connectionString("{app-config-connection-string}")
-    .buildClient();
+OpenTelemetry openTelemetry = otelSdkBuilder.build().getOpenTelemetrySdk();
+Tracer tracer = openTelemetry.getTracer("Sample");
+
+Span span = tracer.spanBuilder("spanName").startSpan();
 
-Span span = tracer.spanBuilder("user-parent-span").startSpan();
-final Scope scope = span.makeCurrent();
-try {
-    // Thread bound (sync) calls will automatically pick up the parent span and you don't need to pass it explicitly.
-    client.setConfigurationSetting("hello", "text", "World");
+// Make the span the current span
+try (Scope scope = span.makeCurrent()) {
+    // Your application logic here
+} catch (Throwable t) {
+    span.recordException(t);
+    throw t;
 } finally {
     span.end();
-    scope.close();
 }
 ```
+The following example demonstrates how to add a span processor to the OpenTelemetry SDK autoconfiguration.
+
+```java readme-sample-span-processor
+AutoConfiguredOpenTelemetrySdkBuilder sdkBuilder = AutoConfiguredOpenTelemetrySdk.builder();
+
+AzureMonitor.customize(sdkBuilder);
+
+SpanProcessor spanProcessor = new SpanProcessor() {
+    @Override
+    public void onStart(Context context, ReadWriteSpan span) {
+        span.setAttribute(AttributeKey.stringKey("attributeKey"), "attributeValue");
```

Comment:
I'd suggest extracting the `AttributeKey.stringKey("attributeKey")` into a constant to encourage that best practice (otherwise you could use the span.setAttribute(String, String) method)